ui/src/wizard/ResourceGroupCreateStep.ts

```diff
@@ -42,7 +42,11 @@ export class ResourceGroupCreateStep<T extends types.IResourceGroupWizardContext
             } else {
                 // if we suspect that this is a Concierge account, only pick the rg if it begins with "learn" and there is only 1
                 if (/concierge/i.test(wizardContext.subscriptionDisplayName)) {
-                    const rgs = await resourceClient.resourceGroups.list();
+                    const rgs: ResourceGroup[] = [];
+                    for await (const rg of resourceClient.resourceGroups.list()) {
```

Comment:
Is `listAllIterator` an option here? Its implementation looks more or less like this.

Reply:
Oh yes, it totally is.  I made that helper function after I started doing it in multiple places and forgot to convert this back 👌